src/input_xml.F90

```diff
@@ -1049,8 +1049,9 @@ subroutine read_geometry_xml()
     do i = 1, n_cells
       c => cells(i)
 
-      ! Initialize the number of cell instances - this is a base case for distribcells
+      ! Initialize distribcell instances and distribcell index
```

Comment:
Yeah, it confuses me too :(  I would really like to completely overhaul distribcell, but it's another of those things that I cannot prioritize very highly right now.